services/azblob/internal/cmd/meta.go

```diff
@@ -0,0 +1,135 @@
+package main
+
+import (
+	def "go.beyondstorage.io/v5/definitions"
+	"go.beyondstorage.io/v5/types"
+)
+
+var Metadata = def.Metadata{
+	Name: "azblob",
+	Pairs: []def.Pair{
+		pairAccessTier,
+		pairEncryptionKey,
+		pairEncryptionScope,
+	},
+	Infos: []def.Info{
+		infoObjectMetaAccessTier,
+		infoObjectMetaEncryptionKeySha256,
+		infoObjectMetaEncryptionScope,
+		infoObjectMetaServerEncrypted,
+	},
+	Factory: []def.Pair{
+		def.PairCredential,
+		def.PairEndpoint,
+		def.PairName,
+		def.PairWorkDir,
+	},
+	Service: def.Service{
+		Features: types.ServiceFeatures{
+			Create: true,
+			Delete: true,
+			Get:    true,
+			List:   true,
+		},
+	},
+	Storage: def.Storage{
+		Features: types.StorageFeatures{
+			VirtualDir:       true,
+			WriteEmptyObject: true,
+
+			Create:       true,
+			CreateDir:    true,
+			CreateAppend: true,
+			Delete:       true,
+			List:         true,
+			Metadata:     true,
+			Read:         true,
+			Stat:         true,
+			Write:        true,
+			WriteAppend:  true,
```

Comment:
We need to mark `CommitAppend` as `true` here because `azblob` does support `CommitAppend`. Or, we can remove the `Appender` test for azblob.

Reply:
> s `true` here

So how about do issue 1022 first? Or I just mark `CommitAppend` as `true` first.